tests/test_fields.py

```diff
@@ -1,13 +1,5 @@
-from __future__ import absolute_import
-
-# Django 1 vs. 2
-try:
-    from django.core.urlresolvers import reverse
-except ImportError:
-    from django.urls import reverse
-
+from django.urls import reverse
```

Comment:
Alphabetisation is off here too - would you mind?

Reply:
Good eye! Fixing